### PR TITLE
Canton AWS Quickstart

### PR DESCRIPTION
**Pull request by davidpadbury-da**
AWS Quickstart for deploying a multi-node Canton topology using CloudFormation. Largely as supplied but have uploaded the template and packaged lambda functions to a public `canton-aws-quickstart` bucket we own which allows users to use this with just the template URL.

**For future changes:**
 * I'll subsequently have to add support for targeting different versions of Canton and versioning the lambda functions accordingly (currently everything assumes canton-community 0.22.0).
 * Should add a basic build step for the lambdas too.

### CODE REVIEW

File: deployment/aws/lambda-functions/create-dbs/package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "canton-aws-quickstart-create-dbs",
+  "version": "1.0.0",
+  "description": "AWS lambda function for creating databases for the quickstart deployment",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Canton Team <team@canton.io>",
```
Comment (daravep) [inline deployment/aws/lambda-functions/create-dbs/package.json:9]: Does that email address work? Normally it's canton@digitalasset.com

**Review by daravep — APPROVED** [review]
Left minor comments.

File: deployment/aws/README.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+## Purpose
+
+This presents an example AWS CloudFormation deployment template for deploying a multi-node Canton topology to AWS.
+
+[![Canton Deployment on AWS](images/canton-deployment-on-aws.svg)](https://lucid.app/lucidchart/6a3c1fb2-f58c-4d5d-bd0f-1b9153b60dd7/edit?beaconFlowId=4585C3F48181BDE0&page=0_0#)
+
+ * Three Canton Participants and two Canton Domains are connected.
+ * Participant1 and Participant2 is connected to Domain1.
+ * Participant2 and Participant3 is connected to Domain1.
```
[the remark this answers is not in the document]
Comment (davidpadbury-da) [reply deployment/aws/README.md:11]: fixed.

File: deployment/aws/README.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+ * Each Participant is fronted by an instance of [DAML Navigator](https://docs.daml.com/tools/navigator/index.html) to allow exploring their view of the Virtual Global Ledger.
+ * Each node has its own database but all are hosted on the same PostgreSQL instance (each application has a dedicated user only permitting access to their database).
+
+To allow generating configuration and creating databases based on values from the CloudFormation template parameters they are provisioned using [AWS Lambda Functions](https://aws.amazon.com/lambda/) during the deployment.
+
+## Limitations
+
+ * All Canton APIs are exposed publicly without authentication.
```
[the remark this answers is not in the document]
Comment (davidpadbury-da) [reply deployment/aws/README.md:19]: There's no easy way to get generated certs to the containers and the user deploying afaik.

File: deployment/aws/README.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+Upload the desired dar file from any project with:
+
+`daml ledger upload-dar --host <ledger_api_host> --port <ledger_api_port> <path_to_dar_file>`
+
+This step will have to be repeated for any nodes that should be in communication with one another.
+
+### JSON API
+
+In order to interact with the models in an easy and reliable fashion, we can use DAMLs JSON API, like so:
```
[the remark this answers is not in the document]
Comment (davidpadbury-da) [reply deployment/aws/README.md:80]: done

File: deployment/aws/lambda-functions/create-dbs/package.json (hunk 1 above)
Comment (davidpadbury-da) [reply deployment/aws/lambda-functions/create-dbs/package.json:9]: ah right. sorry, was thinking of our twitter handle.

**Pull request merged**